src/index.js

```diff
@@ -97,10 +99,7 @@ const ApolloClientPlugin = createPlugin({
 
         return forward(operation);
       });
-      const links = [authMiddleware, connectionLink];
-      if (apolloLink) {
-        links.unshift(apolloLink);
-      }
+      const links = getApolloLinks([authMiddleware, connectionLink]);
```

Comment:
Need to check `if getApolloLinks && typeof(getApolloLinks) === 'function' ? ... : `

Reply:
I am not sure we do, because of the function default argument.
```
getApolloLinks = links => links,
```

Reply:
@KevinGrandon Ah, ok. Then everything is fine.